include/mesh/mesh_function.h

```diff
@@ -159,6 +168,25 @@ class MeshFunction : public FunctionBase<Number>,
                    const std::set<subdomain_id_type> * subdomain_ids);
 
   /**
+   * Similar to operator() with the same parameter list, but with the difference
+   * that multiple values on faces are explicitly permitted. This is useful for
+   * dicontinuos shape functions that are evaluated on faces.
+   */
+  void discontinuousValue (const Point & p,
+                           const Real time,
+                           std::map<const Elem *, DenseVector<Number> > & output);
+
+  /**
+   * Similar to operator() with the same parameter list, but with the difference
+   * that multiple values on faces are explicitly permitted. This is useful for
+   * dicontinuos shape functions that are evaluated on faces.
```

Comment:
typo: "dicontinuos" seems to have been copy/pasted from somewhere else a couple of times.